tools/git-change-exec/actions-lint.go

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type lintSpdx struct {
+	extsMap    map[string]func(path string)
+	pathsToFix []string
+}
+
+func (s *lintSpdx) dontfix(path string) {
+	log.Printf("Cannot fix %s ...\n", path)
+}
+
+func (s *lintSpdx) yamlfix(path string) {
+	license := []string{
+		"# Copyright (c) 2024 Zededa, Inc.\n",
+		"# SPDX-License-Identifier: Apache-2.0\n\n",
+	}
+
+	log.Printf("Fixing %s ...\n", path)
+	prepend(path, license)
+}
+
+func (s *lintSpdx) gofix(path string) {
+	license := []string{
+		"// Copyright (c) 2024 Zededa, Inc.\n",
```

Comment:
The year should not be hardcoded.
And we cannot put Zededa as it's the lf-edge-wide code... (we still can have such a tool in the zededa/eve-tools repo)

Reply:
Agreed on both points. Check for current year and no check on the holder of the copyright.

Reply:
The year is now the current year and the organization is read from the .gitconfig.

As the check is now using the spdx shell script, it does not check on the holder for the copyright.